packages/calcite-components/src/components/block/block.tsx

```diff
@@ -237,17 +237,19 @@ export class Block
   }
 
   renderIcon(): VNode[] {
-    const { el, loading, messages, status } = this;
+    const { loading, messages, status } = this;
 
     const statusIcon = ICONS[status];
 
-    const hasIcon = getSlotted(el, SLOTS.icon) || statusIcon;
+    const loaderIcon = (
+      <div class={CSS.icon}>
+        <calcite-loader inline label={messages.loading} />
+      </div>
+    );
 
-    const iconEl = !statusIcon ? (
-      <slot key="icon-slot" name={SLOTS.icon} />
-    ) : loading ? (
-      <calcite-loader inline label={messages.loading} />
-    ) : (
+    return loading ? (
+      loaderIcon
```

Comment:
Can you inline `loaderIcon`'s JSX here? Otherwise, under the hood, it's creating a `VNode` that may not get used in most cases.

Reply:
It's the linter's preference to have it wrapped.